templates/api.html

```diff
@@ -1,5 +1,11 @@
+<head>
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.8.0/styles/default.min.css">
+<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.8.0/highlight.min.js"></script>
```

Comment:
We have it in the layout.
https://github.com/cdnjs/new-website/blob/master/templates/layout.html#L163

BTW, please always use https and SRI.